common/mqtt_client/tests/packet_size_tests.rs

```diff
@@ -14,9 +15,8 @@ enum TestJoinError {
 // This checks the mqtt packets are within the limit or not
 async fn packet_size_within_limit() -> Result<(), anyhow::Error> {
     // Start the local broker
-    let mqtt_server_handle = tokio::spawn(async {
-        start_broker_local("../../configuration/rumqttd/rumqttd_5883.conf").await
-    });
+    let _mqtt_server_handle =
+        tokio::spawn(async { rumqttd_broker::start_broker_local(58584).await });
```

Comment:
Why each test start it's own broker? Why not reuse the same port for all tests?

Reply:
Because they run in parallel.

Reply:
The tests should be able to share the broker as long as they use different topics.